PowerShell/ScubaGear/Modules/Support/Support.psm1

```diff
@@ -250,7 +250,7 @@ function Install-OPAforSCuBA {
     )
 
     # Constants
-    $ACCEPTABLEVERSIONS = [ScubaConfig]::ScubaDefault('DefaultOPAVersion') # End Versions
+    $ACCEPTABLEVERSIONS = '0.69.0', [ScubaConfig]::ScubaDefault('DefaultOPAVersion') # End Versions
```

Comment:
Why is 0.69.0 explicitly added as an acceptable version?

Reply:
Usually the previous supported version (0.69.0 in this case) is appended to this variable. @buidav Why did we remove the list of acceptable versions from the [prior OPA bump to v0.69.0?](https://github.com/cisagov/ScubaGear/pull/1348/files)

Reply:
Ah that's why, didn't scroll down far enough to read that. Thanks.

Reply:
That makes sense, everything ran correctly when I tested.